web/frontend/src/components/arm.vue

```diff
@@ -39,11 +39,21 @@ const fieldMap = [
   ['y', 'y'],
   ['z', 'z'],
   ['theta', 'theta'],
-  ['o_x', 'ox'],
-  ['o_y', 'oy'],
-  ['o_z', 'oz'],
+  ['o_x', 'oX'],
+  ['o_y', 'oY'],
+  ['o_z', 'oZ'],
 ] as const;
 
+const updateFieldMap = {
+  X: 'x',
+  Y: 'y',
+  Z: 'z',
+  Theta: 'theta',
+  OX: 'oX',
+  OY: 'oY',
+  OZ: 'oZ',
+} as const;
```

Comment:
sigh... this is a mess - probably worth a more holistic cleanup the next time we update the arm card.